import/popups/MicrophoneDialog.qml

```diff
@@ -0,0 +1,55 @@
+/****************************************************************************
+**
+** Copyright (c) 2013 - 2016 Jolla Ltd.
+** Copyright (c) 2020 - 2021 Open Mobile Platform LLC.
```

Comment:
Initially the file was copied from the neighbor LocationDIalog.qml, it has Jolla copyright. But since then, the content has completely changed, so I left only OMP's. Thanks!